pkg/controller/volume/persistentvolume/pv_controller.go

```diff
@@ -955,10 +955,8 @@ func (ctrl *PersistentVolumeController) assignDefaultStorageClass(ctx context.Co
 
 	class, err := util.GetDefaultClass(ctrl.classLister)
 	if err != nil {
-		// It is safe to ignore errors here because it means we either could not list SCs or there is more than one default.
-		// TODO: do not ignore errors after this PR is merged: https://github.com/kubernetes/kubernetes/pull/110559
-		logger.V(4).Info("Failed to get default storage class", "err", err)
-		return false, nil
+		logger.Error(err, "Failed to get default storage class")
```

Comment:
The error will be logged in `updateClaim()`, no need to log it here.

Reply:
Correct, removing.